mod/notifications/start.php

```diff
@@ -41,6 +41,10 @@ function notifications_plugin_init() {
 function notifications_page_handler($page) {
 
 	elgg_gatekeeper();
+
+	// Set the context to settings
+	elgg_set_context('settings');
+
```

Comment:
I'm for keeping logic in the resource views and having mostly dumb page handlers that do very little. To me it's worth the duplication to have all the logic in the page resource view. It would also ease the transition to a modern routing system.